### PR TITLE
Prepare environment for further ET deployments

### PR DESCRIPTION
**Pull request by zuzueeka**
1. Add template with structure for ET payment setup addresses
2. Add Sandbox setup for stables addresses (contracts were deployed by @bulbozaur), deployment tx: https://holesky.etherscan.io/tx/0x1317683f5559873be95aab298cd4bd2584468075082f9a1b2e9c037a9b329946
3. Add Stables addresses (contracts were deployed by @bulbozaur), used for allowedTokensRegistry
4. Add DAO-ops & QA ms (planned to be used as a trustCaller for payment setups)

### CODE REVIEW

**Review by TheDZhon — COMMENTED** [review]
Like the proposed layout :+1:  

though can't approve without the complete address set.

File: docs/deployed-contracts/holesky.md [hunk 1]
```diff
@@ -81,12 +81,60 @@
...
+  - AllowedRecipientsBuilder (multi token): [`0x983dF2EA3A7Dce9D60bD06f5C5dCc44a138eBA89`](https://holesky.etherscan.io/address/0x983dF2EA3A7Dce9D60bD06f5C5dCc44a138eBA89)
+  - AllowedRecipientsFactory (multi token): [`0x9d156F7F5ed1fEbDc4996CAA835CD964A10bd650`](https://holesky.etherscan.io/address/0x9d156F7F5ed1fEbDc4996CAA835CD964A10bd650)
+  - BokkyPooBah's DateTime Library: [`0xd6237FecDF9C1D9b023A5205C17549E3037EeEec`](https://holesky.etherscan.io/address/0xd6237FecDF9C1D9b023A5205C17549E3037EeEec)
 
+## Lido DAO Multisigs
+  - QA & DAO-ops ms: [`0x96d2Ff1C4D30f592B91fd731E218247689a76915`](https://stg.holesky-safe.protofire.io/home?safe=holesky:0x96d2Ff1C4D30f592B91fd731E218247689a76915)
+    - QA testnet EOA: [`0x1580881349e214Bab9f1E533bF97351271DB95a9`](https://holesky.etherscan.io/address/0x1580881349e214Bab9f1E533bF97351271DB95a9)
+    - DAO-ops testnet EOA: [`0xd4090CA1134F8dE1450B8246916F73d212efdEf6`](https://holesky.etherscan.io/address/0xd4090CA1134F8dE1450B8246916F73d212efdEf6)
+## Stablecoins
```
Comment (TheDZhon) [inline docs/deployed-contracts/holesky.md:137]:
```suggestion
## Testnet Stablecoins
```

File: docs/deployed-contracts/holesky.md [hunk 2]
```diff
@@ -81,12 +81,60 @@
...
+  - RemoveAllowedRecipient: [`0x51c730af05777c4D3CcC8c8B80558F4D155bb7BF`](https://holesky.etherscan.io/address/0x51c730af05777c4D3CcC8c8B80558F4D155bb7BF)
+- **Tooling contracts:**
+  - AllowedRecipientsBuilder (single token): [`0xeC3785b13b21c226D66B5bC2E82BB2f4226f715e`](https://holesky.etherscan.io/address/0xeC3785b13b21c226D66B5bC2E82BB2f4226f715e)
+  - AllowedRecipientsFactory (single token): [`0x62a65b0eC11D74a00FcC445ef7A3374f02635Dd9`](https://holesky.etherscan.io/address/0x62a65b0eC11D74a00FcC445ef7A3374f02635Dd9)
+  - AllowedRecipientsBuilder (multi token): [`0x983dF2EA3A7Dce9D60bD06f5C5dCc44a138eBA89`](https://holesky.etherscan.io/address/0x983dF2EA3A7Dce9D60bD06f5C5dCc44a138eBA89)
+  - AllowedRecipientsFactory (multi token): [`0x9d156F7F5ed1fEbDc4996CAA835CD964A10bd650`](https://holesky.etherscan.io/address/0x9d156F7F5ed1fEbDc4996CAA835CD964A10bd650)
+  - BokkyPooBah's DateTime Library: [`0xd6237FecDF9C1D9b023A5205C17549E3037EeEec`](https://holesky.etherscan.io/address/0xd6237FecDF9C1D9b023A5205C17549E3037EeEec)
 
+## Lido DAO Multisigs
```
Comment (TheDZhon) [inline docs/deployed-contracts/holesky.md:133]:
```suggestion
## Testnet DAO Multisigs
```

**Review by bulbozaur — APPROVED** [review]
LGTM

**Pull request merged**